backend/battles/migrations/0007_auto_20210113_1745.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.17 on 2021-01-13 17:45
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    Battle = apps.get_model("battles", "Battle")
+    BattleTeam = apps.get_model("battles", "BattleTeam")
+
+    db_alias = schema_editor.connection.alias
```

Comment:
Why do we need this?

Reply:
Oh, sorry. This used in this Django documentation example: https://docs.djangoproject.com/en/3.1/ref/migration-operations/#django.db.migrations.operations.RunPython
But that is helpful when we have multiple databases, which is not the case here. I'll remove it.